packages/design-system/src/components/Dropdown/Dropdown.stories.jsx

```diff
@@ -23,25 +26,38 @@ const dropdownOptions = [
   { label: 'Option 8', value: '8' },
 ];
 
-const optGroupData = (
+const htmlOptGroups = (
   <>
-    <optgroup label="Option group">
-      <option value="1-1">Option 1</option>
-      <option value="1-2">Option 2</option>
-      <option value="1-3">Option 3</option>
+    <optgroup label="Group A">
+      <option value="a-1">Option A-1</option>
+      <option value="a-2">Option A-2</option>
+      <option value="a-3">Option A-3</option>
     </optgroup>
-    <optgroup label="More option groups">
-      <option value="2-1">Option 4</option>
-      <option value="2-2">Option 5</option>
-      <option value="2-3">Option 6</option>
+    <optgroup label="Group B">
+      <option value="b-1">Option B-1</option>
+      <option value="b-2">Option B-2</option>
+      <option value="b-3">Option B-3</option>
     </optgroup>
   </>
 );
 
+const htmlOptions = (
```

Comment:
Are these "html" prefixed stories supposed to have native HTML inside this select? Running it locally and I'm not seeing this markup - it's all lists.

Reply:
It accepts HTML but outputs the same thing as the non-HTML examples. It parses child elements and converts it into options that can be fed to our Downshift-based listbox